sdk/cosmos/azure-cosmos/pom.xml

```diff
@@ -104,7 +104,12 @@ Licensed under the MIT License.
       <scope>test</scope>
       <version>4.2</version> <!-- {x-version-update;org.apache.commons:commons-collections4;external_dependency} -->
     </dependency>
-
+    <!-- https://mvnrepository.com/artifact/org.hdrhistogram/HdrHistogram -->
+    <dependency>
+      <groupId>org.hdrhistogram</groupId>
+      <artifactId>HdrHistogram</artifactId>
+      <version>2.1.4</version>
```

Comment:
We need to tag it properly through external_dependencies.txt file - see the above external dependencies for examples on how to tag it.

Reply:
done